app/src/main/java/chat/rocket/android/widget/DividerItemDecoration.kt

```diff
@@ -56,10 +57,13 @@ class DividerItemDecoration() : RecyclerView.ItemDecoration() {
             if (isLastView(child, parent))
                 continue
 
+            if (isViewTypeHeader(child, parent))
```

Comment:
You don't need to add another conditional here. Just define it:

```
if (isLastView(child, parent) || isViewTypeHeader(child, parent)) 
   continue
```

Reply:
@filipedelimabrito  sorry about that :man_facepalming: